e2e-tests/tfm/cypress/e2e/journeys/utilisation-reports/report-reconciliation/bank-utilisation-report-page.spec.js

```diff
@@ -38,14 +38,52 @@ context('Bank utilisation report page', () => {
     cy.task(NODE_TASKS.INSERT_TFM_FACILITIES_INTO_DB, matchingTfmFacilities);
   });
 
-  beforeEach(() => {
-    pages.landingPage.visit();
-    cy.login(USERS.PDC_RECONCILE);
+  describe(`when navigating to the page as ${USERS.PDC_RECONCILE}`, () => {
+    beforeEach(() => {
+      pages.landingPage.visit();
+      cy.login(USERS.PDC_RECONCILE);
 
-    cy.visit(`utilisation-reports/${reportId}`);
+      cy.visit(`utilisation-reports/${reportId}`);
+    });
+
+    it('should display the page heading and date', () => {
+      cy.assertText(utilisationReportPage.heading(), bankName);
+
+      cy.assertText(utilisationReportPage.reportPeriodHeading(), reportPeriodString);
+    });
+
+    it('should display the correct tabs', () => {
+      cy.assertText(utilisationReportPage.premiumPaymentsTabLink(), 'Premium payments');
+      cy.assertText(utilisationReportPage.keyingSheetTabLink(), 'Keying sheet');
+      cy.assertText(utilisationReportPage.paymentDetailsTabLink(), 'Payment details');
+      cy.assertText(utilisationReportPage.utilisationTabLink(), 'Utilisation');
+    });
```

Comment:
Same as above - are we testing the hrefs somewhere else?

If not we could split each tab into it's own `it` and assert the text and href

Reply:
no hrefs, these are rending html

Reply:
Resolving as there is another comment RE this